editor/animation_track_editor.cpp

```diff
@@ -6479,6 +6539,7 @@ AnimationTrackEditor::AnimationTrackEditor() {
 	timeline->connect("length_changed", callable_mp(this, &AnimationTrackEditor::_update_length));
 
 	panner.instantiate();
+	panner->set_scroll_zoom_factor(1.02);
```

Comment:
```suggestion
	panner->set_scroll_zoom_factor(SCROLL_ZOOM_FACTOR);
```
Replace 1.02 in BezierEditor as well.

Reply:
woops, I missed that one, sorry. Resolved.